src/routes/+layout.server.ts

```diff
@@ -35,6 +35,7 @@ export async function load(event) {
 		websiteShortDescription:
 			runtimeConfig[`translations.${event.locals.language}.config`]?.websiteShortDescription ||
 			runtimeConfig.websiteShortDescription,
-		viewportWidth
+		viewportWidth,
+		notResponsive
```

Comment:
move this to `(app)/+layout.server.ts` since it's only needed for UI-rendered pages